.takt/runs/20260217-062316-distinct-distinctbyオペレーター-ha/logs/reports-history/summary.20260217T064615Z.md

```
@@ -0,0 +1,104 @@
+- **空間計算量:** O(k)（k = ユニークな要素数/キー数）
+- **メモリ使用:** ストリーム終了まで `BTreeSet` に全ユニーク要素を保持
+
+## 確認コマンド
+
+### ビルド確認
+```bash
+cargo build -p fraktor-streams-rs
+```
+**結果:** ✅ 成功 (0.10s)
+
+### テスト確認
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)